### DISCUSSION TITLE
Roadmap para Technical Writing?

### DISCUSSION THREAD

**Discussion by dpPdro** [pedroar9/technical-writing#8] [Q&A] [answered]
Qual seria um bom roadmap para Technical Writing?

**Comment by pedroar9** [comment, marked as answer]
Site com Roadmap para Technical Writing.

https://roadmap.sh/technical-writer

technical-writer.pdf

**Comment by pedroar9** [comment]
ROADMAP

- Entenda o que é Technical Writing
- Compreenda a diferença entre Technical Writing e UX Writing
- Conheça boas práticas para ajudar na elaboração de documentações
- Conheça os principais tipos de documentação técnica no contexto de TI
- Entenda os princípios da arquitetura da informação aplicados a TW
- Aprenda a desenvolver documentações como: Release notes, Troubleshooting, tutoriais e mais
- Aprenda a planejar e analisar documentações
